### PR TITLE
Enriches listing models use case scenario

### PR DESCRIPTION
**Pull request by waldemarnt**
I just added a request flow that shows how to discover
and equipment brand just navigating through the links.
After:
screen shot 2016-08-10 at 11 49 54 am

### CODE REVIEW

File: source/includes/use_case_scenarios/_listing_models.md [hunk 1]
```diff
@@ -29,3 +29,23 @@ a list of them using the HTTP method `GET`.
 4. The last step of the flow has to list models of an `equipment type` and `series`. Using the `series`
 identifier and `equipment types` identifier it is possible to make a request to [models](#models-request-parameters)
 resource including the `links` parameters that will filter `models` by `series` and `equipment types` using their identifiers.
+
+### Finding the Equipment Brand through the links
+
+As is it possible to find models just filtering by their links is also possible to discover a `Brand` of an Equipment
```
Comment (lucaslago) [inline source/includes/use_case_scenarios/_listing_models.md]: s/As is it possible/As it is possible

Comment (lucaslago) [inline source/includes/use_case_scenarios/_listing_models.md]: s/links is/links it is

File: source/includes/use_case_scenarios/_listing_models.md [hunk 2]
```diff
@@ -29,3 +29,23 @@ a list of them using the HTTP method `GET`.
 4. The last step of the flow has to list models of an `equipment type` and `series`. Using the `series`
 identifier and `equipment types` identifier it is possible to make a request to [models](#models-request-parameters)
 resource including the `links` parameters that will filter `models` by `series` and `equipment types` using their identifiers.
+
+### Finding the Equipment Brand through the links
```
Comment (carolinaborim) [inline source/includes/use_case_scenarios/_listing_models.md]: ### Finding Equipment Brand through the links

File: source/includes/use_case_scenarios/_listing_models.md [hunk 3]
```diff
@@ -29,3 +29,23 @@ a list of them using the HTTP method `GET`.
...
+doing the same way.
+
+### Request flow
+
+The request flow below shows how to find a Brand of an Equipment navigating through the links.
+
+1. The first request will find an [Equipment]() using the equipment resource on `Telemetry API`.
+
+2. The returned Equipment will be linked with a `Model`. The next request will be done to the
+[model]() resource on `Telemetry API` passing the `model id` as parameter.
```
Comment (lucaslago) [inline source/includes/use_case_scenarios/_listing_models.md]: `[model]()` is this empty link intended?

File: source/includes/use_case_scenarios/_listing_models.md [hunk 4]
```diff
@@ -29,3 +29,23 @@ a list of them using the HTTP method `GET`.
...
+
+3. The model returned from the previous request will be linked with `Series`. The next request will be done to the
+[series]() resource  passing the `series id` as parameter.
```
Comment (lucaslago) [inline source/includes/use_case_scenarios/_listing_models.md]: `[model]()` is this empty link intentional?

File: source/includes/use_case_scenarios/_listing_models.md [hunk 5]
```diff
@@ -29,3 +29,23 @@ a list of them using the HTTP method `GET`.
...
+
+4. The returned series will be linked with `Brands`. The last request will be done to the
+[brands]() resource passing the `brand id` as parameter.
```
Comment (lucaslago) [inline source/includes/use_case_scenarios/_listing_models.md]: `[brands]()` is this empty link intentional?

Comment (waldemarnt) [inline source/includes/use_case_scenarios/_listing_models.md]: thanks, I just forget that.

**Comment by lucaslago** [comment]
👍

**Comment by carolinaborim** [comment]
🐑  🇮🇹

**Pull request merged**